project/dexter-vd-cpp/src/java/com/samsung/sec/dexter/vdcpp/checkerlogic/MagicNumberDataType64BitCheckerLogic.java

```diff
@@ -0,0 +1,131 @@
+import org.eclipse.cdt.core.dom.ast.IASTFunctionDefinition;
+import org.eclipse.cdt.core.dom.ast.IASTIdExpression;
+import org.eclipse.cdt.core.dom.ast.IASTLiteralExpression;
+import org.eclipse.cdt.core.dom.ast.IASTName;
+import org.eclipse.cdt.core.dom.ast.IASTNode;
+import org.eclipse.cdt.core.dom.ast.IASTTranslationUnit;
+import org.eclipse.cdt.core.dom.ast.IBinding;
+import org.eclipse.cdt.core.dom.ast.IVariable;
+import org.eclipse.cdt.core.dom.ast.IType;
+import org.eclipse.cdt.core.dom.ast.IBasicType;
+import org.eclipse.cdt.core.dom.ast.IValue;
+import org.eclipse.cdt.core.dom.ast.IASTEqualsInitializer;
+import org.eclipse.cdt.core.dom.ast.IASTStatement;
+import org.eclipse.cdt.core.dom.ast.IASTReturnStatement;
+import org.eclipse.cdt.core.dom.ast.IASTDeclSpecifier;
+import org.eclipse.cdt.core.dom.ast.IASTSimpleDeclSpecifier;
+import org.eclipse.cdt.core.dom.ast.IASTNamedTypeSpecifier;
+
+import com.samsung.sec.dexter.core.analyzer.AnalysisConfig;
+import com.samsung.sec.dexter.core.analyzer.AnalysisResult;
+import com.samsung.sec.dexter.core.checker.Checker;
+import com.samsung.sec.dexter.vdcpp.plugin.DexterVdCppPlugin;
+import com.samsung.sec.dexter.vdcpp.util.CDTASTUtil;
+
+
+public class MagicNumberDataType64BitCheckerLogic implements ICheckerLogic{
+
+	private IASTTranslationUnit translationUnit;
+	private final static Logger LOG = Logger.getLogger(DexterVdCppPlugin.class);
+
+	@Override
+	public void analyze(final AnalysisConfig config, final AnalysisResult result, 
+			final Checker checker, final IASTTranslationUnit unit) {
+		translationUnit =unit;
+		
+		ASTVisitor visitor = createVisitor(config, result, checker);
+		visitor.shouldVisitDeclarations = true;
+		
+		unit.accept(visitor);	
+	}
+
+	private ASTVisitor createVisitor(final AnalysisConfig config,
+			final AnalysisResult result, final Checker checker) {
+		ASTVisitor visitor = new ASTVisitor() {
+			@Override
+			public int visit(IASTDeclaration ast ) {					
+
+				visitFunction(config, result, checker, ast);
+				
+				return super.visit(ast);
+			}
+
+			private void visitFunction(final AnalysisConfig config,
+					final AnalysisResult result, final Checker checker,
+					final IASTDeclaration ast) {				
+				if(!CDTASTUtil.isFunctionDefinition(ast))	
+					return;					
+
+				if(!CDTASTUtil.checkReturnType((IASTFunctionDefinition)ast,"size_t"))	
+					return;
+				
+				ASTVisitor visitor = new ASTVisitor() {
+					public int visit(IASTStatement statement) {
+						visitReturnStatement(config, result, checker, statement);
+								
+						return ASTVisitor.PROCESS_CONTINUE;
+					}
+
+					private void visitReturnStatement(
+							final AnalysisConfig config,
+							final AnalysisResult result, final Checker checker,
+							IASTStatement statement) {	
+						if(!CDTASTUtil.isReturnStatement(statement))
+							return;
+						
+						if(!isMagicNumberAssignedToReturnValue((IASTReturnStatement)statement, "0xffffffff"))
```

Comment:
Make "0xffffffff" as a contant string(Class member field).
for example,
final static String MAX_INT_VALUE = "0xfffffffff";